Source/src/core/Scene.cpp

```diff
@@ -86,6 +86,22 @@ Hachiko::GameObject* Hachiko::Scene::LoadFBX(const std::string& path)
     return model;
 }
 
+Hachiko::GameObject* Hachiko::Scene::LoadImageObject(const std::string& path)
+{
+    const auto file_name = path.substr(path.find_last_of("/\\") + 1);
```

Comment:
using std::filesystem::path will allow us to get useful information such as the filename and the extension
[CPP Reference](https://en.cppreference.com/w/cpp/filesystem/path/filename)